pkg/rancher-desktop/assets/translations/en-us.yaml

```
@@ -4742,6 +4742,15 @@ snapshots:
     icon: icon-search
     heading: No snapshots found
     body: Click on Create Snapshot to get started
+  info:
+    restore:
+      success: '{snapshot} was successfully restored'
+      cancel: 'You successfully cancelled the restoration of {snapshot}'
+    create:
+      success: '{snapshot} was successfully created and added to the list'
```

We might be able to distill this down to just 

```suggestion
      success: 'Created {snapshot}'
```

I don't think that we need to mention that it was added to the list, unless we foresee scenarios where a snapshot might be created without being added to the list of snapshots.

---

+1 on this. Because we sort in descending order based on creation time, the new snapshot will always appear at the top of the list. 15 years ago someone would have done a yellow-fade around the new snapshot to highlight that it was added. Is there a more contemporary way of highlighting a change and then getting out of the way so the new entry blends in with the existing ones?

---

It make sense, We could put all possible UX improvements in the later release.